autoware_control_performance_analysis/FloatStamped longitudinal_acceleration
autoware_control_performance_analysis/FloatStamped longitudinal_jerk
autoware_control_performance_analysis/FloatStamped lateral_acceleration
autoware_control_performance_analysis/FloatStamped lateral_jerk
autoware_control_performance_analysis/FloatStamped desired_steering_angle
autoware_control_performance_analysis/FloatStamped controller_processing_time
